src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -58,10 +60,10 @@ public abstract class SkriptEventHandler {
 	 * the {@link EventExecutor} to be used with this listener.
 	 */
 	public static class PriorityListener implements Listener {
-		
+
 		public final EventPriority priority;
 		public final Set<Class<? extends Event>> registeredEvents = new HashSet<>();
```

Comment:
The only reason for the registeredEvents field is to check which events have been registered, so you can remove it